release.yaml

```diff
@@ -247,3 +247,47 @@ promoteToPublic:
           cat << EOF | buildkite-agent annotate --style info
           Promoted release is **publicly available** through a git tag at [\`{{version}}\`](https://github.com/sourcegraph/deploy-sourcegraph-docker/tree/{{version}}).
           EOF
+
+      - name: "update main with latest version"
+      - name: "git"
+        cmd: |
+          set -eu
+          branch="promote/release-{{version}}-update-main"
+          echo "Checking out origin/main"
+          git fetch origin main
+          git switch main
```

Comment:
Same as [previous commentary](https://github.com/sourcegraph/deploy-sourcegraph-helm/pull/600#discussion_r1887535527) in helm repo changes: https://github.com/sourcegraph/deploy-sourcegraph-docker/pull/1124/files#diff-a0b29149f10adefb5c44814777ca3aa3c992d81fd92a1a051add31fab99452fcR223-R225